roles/matrix-bridge-hookshot/templates/config.yml.j2

```diff
@@ -0,0 +1,127 @@
+#jinja2: lstrip_blocks: "True"
+bridge:
+  # Basic homeserver configuration
+  #
+  domain: {{ matrix_domain }}
+  url: {{ matrix_hookshot_homeserver_address }}
+  mediaUrl: {{ matrix_hookshot_homeserver_address }}
+  port: {{ matrix_hookshot_appservice_port }}
+  bindAddress: 0.0.0.0
+{% if matrix_hookshot_github_enabled %}
+github:
+  # (Optional) Configure this to enable GitHub support
+  #
+  auth:
+    # Authentication for the GitHub App.
+    #
+    id: {{ matrix_hookshot_github_appid }}
+    privateKeyFile: /data/{{ matrix_hookshot_github_private_key }}
+  webhook:
+    # Webhook settings for the GitHub app.
+    #
+    secret: {{ matrix_hookshot_github_secret }}
```

Comment:
Using `|to_json` here (and *almost* everywhere else) would be beneficial in cases where the value contains special characters that need to be escaped.

Example:

```yaml
secret: {{ matrix_hookshot_github_secret|to_json }}
```

This secret is probably safe, but it's good practice to use `|to_json` everywhere anyway.

------

On a related note, doing this will not work though:

```diff
-privateKeyFile: /data/{{ matrix_hookshot_github_private_key }}
+privateKeyFile: /data/{{ matrix_hookshot_github_private_key|to_json }}
```

You either need to leave that one as it is, or using something like:

```diff
-privateKeyFile: /data/{{ matrix_hookshot_github_private_key }}
+privateKeyFile: {{ ('/data' + matrix_hookshot_github_private_key) | to_json }}
```